pkgs/by-name/li/libportal/package.nix

```diff
@@ -74,9 +74,9 @@ stdenv.mkDerivation rec {
     (lib.mesonEnable "backend-gtk4" (variant == "gtk4"))
     (lib.mesonEnable "backend-qt5" (variant == "qt5"))
     (lib.mesonEnable "backend-qt6" (variant == "qt6"))
-    (lib.mesonBool "vapi" (variant != "qt5"))
-    (lib.mesonBool "introspection" (variant != "qt5"))
-    (lib.mesonBool "docs" (variant != "qt5")) # requires introspection=true
+    (lib.mesonBool "vapi" (variant != "qt5" && variant != "qt6"))
```

Comment:
Perhaps this should be a separate commit with an explanation in the commit message?